appveyor.yml

```diff
@@ -126,8 +126,7 @@ before_deploy:
   - "SET PATH=%ORIGPATH%"
   - "%CMD_IN_ENV% %WP_INSTDIR%/scripts/env.bat"
   - "%CMD_IN_ENV% python.exe setup.py install"
-  - "%CMD_IN_ENV% easy_install --upgrade pip"
-  - "%CMD_IN_ENV% pip install --upgrade configobj traitsui"
+  - "%CMD_IN_ENV% pip install --upgrade configobj traitsui start_jupyter_cm"
```

Comment:
Not really a WinPython issue as we do not use any WinPython tools for creating our bundle. We simply repackage a bundle based on a WinPython installation. I mentioned using the WinPython routines so we don't have to reinvent the wheel.